example/graphbrainz/lib/main.dart

```diff
@@ -23,5 +21,5 @@ Future<void> main() async {
   final edSheeran = response.data.node as Artist;
   print(edSheeran.name);
   print(edSheeran.lifeSpan.begin);
-  print(edSheeran.spotify.href);
+//  print(edSheeran.spotify.href);
```

Comment:
Why was that commented?

Reply:
I was breaking with Null value 🤷‍♂ (probably something in the API changed)

Reply:
meh, just remove it then, please